stablehlo/tests/ops_stablehlo.mlir

```diff
@@ -3540,32 +3540,32 @@ func.func @reduce_precision_invalid_mantissa(%arg: tensor<2x4xf32>) -> tensor<2x
 func.func @gather(%operand : tensor<2x4x9xi32>, %start_indices : tensor<1x5x2xi32>) -> tensor<1x5x8xi32> {
```

Comment:
Do we have a test to check `c13`?

Reply:
It was an oversight to not label this one. Updated the function name in `infer_stablehlo.mlir` which covers this.